lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/mv/XwikiController.java

```diff
@@ -121,15 +125,42 @@ public void notifyVerification(String verificationProcessId)
         String type = this.typesMap.get(modelSetId);
         
 		boolean resultsOk = res.getFinalResult().equals(FinalResultType.OK);
+		boolean importedInTheSimulator = true;
 		if(resultsOk){
 		    if(XWikiRestUtils.isPage(RestResource.CORE_REPOSITORY_WIKI,
 	                RestResource.CORE_REPOSITORY_SPACE, modelSetId) == true){
     		    this.cw.modelSetImported(modelSetId, type);
     		    this.or.modelSetImported(modelSetId, type);
+    		    
+    		    String bpmnFileURL = XWikiRestUtils.exposeBPMNFromCoreRepository(modelSetId, type);
+    		    Collection<String> savedProcessesList = this.sim.addProcessDefinition(bpmnFileURL);
+    		    importedInTheSimulator = this.verifyImportedProcesses(savedProcessesList, bpmnFileURL);
 		    }
 		}
 		
-		this.cw.modelVerified(modelSetId, (resultsOk)?"OK":"KO");
+		// the value for message may change accordingly to eu.learnpad.cw.rest.ModelVerified
+		String message = "KO";
+		if ( resultsOk ){
```

Comment:
1) fine with me to change the name in the API from `cw.modelVerified` to `cw.modelsetVerified`
2) with @damianofalcioni we imaged that such notification could be useful for pratical purposes form the users of the platform. For example it would be nice to have a kind of report page (e.g. a log) which is accessible from the GUI of the CW.